spec/index.html

```diff
@@ -734,9 +735,153 @@ <h3>Overview</h3>
     <section id="hash-1d-quads-exampkes" class="informative">
       <h3>Examples</h3>
 
-      <p class="ednote">
-        Add some examples where the algorithm results are unique or not.
-      </p>
+      <aside id="ex-unique-hashes"
+            class="example"
+            title="Unique hashes">
+        <p>This example illustrates hashing quads containing
+          blank nodes where hashing the statements mentioning
+          those blank nodes generates unique results.</p>
+
+        <figure id="fig-unique-hashes" style="text-align:center">
+          <!-- Source for this file is at https://docs.google.com/drawings/d/1LQ6fp4a35lrEOKRMD20gcncUkoitX0eW8QI_U2mQnAs/edit?usp=sharing -->
+          <object data="unique-hashes.svg" style="width: 75%" type="image/svg+xml" aria-describedby="fig-unique-hashes-alt">
+            <p id="fig-unique-hashes-alt">
+              The image represents the graph described in
+              <a href="#ex-unique-hashes-input">
+                the following code block
+              </a>.</p>
+          </object>
+          <figcaption>An illustration of a graph resulting in unique hashes.<br/>
+            Image available in
+            <a href="unique-hashes.svg" title="SVG image of a unique hashes">
+              <abbr title="Scalable Vector Graphics">SVG</abbr>
+            </a>.</figcaption>
+        </figure>
+
+        <pre id="ex-unique-hashes-input"
+             data-transform="updateExample"
+             title="Graph with blank nodes resulting in unique hashes">
+          <!--
+            :p :q _:b0 .
+            :p :r _:b1 .
+            _:b0 :p _:b2 .
+            _:b1 :p _:b2 .
+          -->
+        </pre>
+
+        <p>The algorithm will be called twice, with each blank node (`_:b0` and `_:b1`).
+          Running the algorithm with the reference node `_:b0` results in the
+          following quads, after replacing blank nodes:</p>
+
+        <pre data-transform="updateExample">
+          <!--
+            :p :q _:a .
+            _:a :p _:z .
+          -->
+        </pre>
+
+        <p>These are then serialized to N-Quads, concatenated and hashed resulting in
+          `3b26142829b8887d011d779079a243bd61ab53c3990d550320a17b59ade6ba36`.</p>
```

Comment:
If this is a real hash, we should probably say that the hash algorithm in use here is SHA-256. At a later time we should do a pass over the spec to make sure we can swap the hash algorithm out for something else in the future (which would create a "new canonicalization algorithm", but only different in the hash algorithm chosen).

Reply:
Would a reference to "hash algorithm", which does say that it is SHA-256 work for you? Or, would you rather replace the actual hash value with something more abstract? My thought was that we should strive to make the examples accurate, so that we could potentially automatically check for correctness.

Reply:
Yeah, a reference would be fine.